### PR TITLE
Undelete wasn't working with new restrictions table.

### PR DESCRIPTION
**Pull request by Pchelolo**
No need to refetch stuff that we've just saved.

cc @wikimedia/services

### CODE REVIEW

File: sys/page_revisions.js [hunk 1]
```diff
@@ -679,10 +681,18 @@ PRS.prototype.getRevision = function(hyper, req) {
             }
         });
     }
+
+    function fetchFromMW() {
```
Comment (gwicke) [inline sys/page_revisions.js]: The naming suggests that this is only fetching, but it is really saving as well. Perhaps something like fetchStoreAndCheckMWRevision could be better? Or, actually, is there a call to fetchAndStoreMWRevision that doesn't want the checking part?

Comment (Pchelolo) [inline sys/page_revisions.js]: Yep, when we're using it from `getTitleRevison` we don't wanna check because it's already checking afterwards.

**Comment by gwicke** [comment]
LGTM. Waiting for travis before merging.

**Comment by d00rman** [comment]
:100:

**Pull request merged**